lib/queuePopulator/IngestionProducer.js

```diff
@@ -383,6 +401,36 @@ class IngestionProducer {
         }
         return done(null, []);
     }
+
+    /**
+     * Get bucket cseq
+     * @param {string} bucket - bucket name
+     * @param {function} done - callback(err, cseq) where `cseq` is an integer
+     * @return {undefined}
+     */
+    _getBucketCseq(bucket, done) {
+        return this._ringReader.getBucketCseq({ Bucket: bucket },
+        (err, data) => {
+            if (err) {
+                this.log.error('error getting bucket cseq', {
+                    method: 'IngestionProducer._getBucketCseq',
+                    error: err,
+                    bucket,
+                });
+                return done(err);
+            }
+            if (!data || !data.length > 0 || !data[0]) {
```

Comment:
`!data.length > 0` is redundant since if the case, `!data[0]` will also return true.